src/index.js

```diff
@@ -0,0 +1,9 @@
+import { instance } from './addAndRemove.js';
+import { dragTask } from './reorder.js';
```

Comment:
When importing, you shouldn't have left extensions of '.js', because linter doesn't allow this. It should work in both cases but I'm interested in how you fixed linter errors without removing them.

Reply:
Actually, if I don't put the extensions (.js) the Linter doesn't allow it. 
>1:26  warning  Missing file extension "js" for "./addAndRemove"  import/extensions
>2:26  warning  Missing file extension "js" for "./reorder"       import/extensions

I saw other student's repositories and they all have the extensions. So now I'm interested in how you fixed linter errors without removing them:thinking: